### ISSUE TITLE
Cannot select stream from Plex APP using PKC Companion feature

### ISSUE THREAD

**Issue by tomzini** [croneter/PlexKodiConnect#411]
when using Plex App with a different user from the one used in PKC, when I play a video content with multiple versions a popup appears on kodi asking me to "select a stream", but is not possible perform the selection

**Comment by croneter** [comment]
Again, https://github.com/croneter/PlexKodiConnect/wiki/How-to-Report-A-Bug

**Comment by tomzini** [comment]
sorry ...
PKC versione 2.0.2

**Comment by croneter** [comment]
Debug log?

**Comment by tomzini** [comment]
Hi,
in the below link the log for  this issue. The log file refer to a test done with last 2.0.3 version
https://pastebin.com/dl/HAhnmhn6

i have tried also popping up the "remote control" from the plex app, but I can't move on screen between various title versions.
i used finally a keyboard to let it start and to finish the test, collecting logs.

**Comment by croneter** [comment]
Sorry, took a while...

I see this stuff at the beginning of your log:
```
PLEX.PlexFunctions: Checking connection to server https://10.0.2.111:32400/library/onDeck with verifySSL=False
## LibreELEC Addon ## oe::load_url(https://mediacenter.nextworks.it/os/updates.php?i=affb7782133b6c272eae2867da078b13&d=LibreELEC&pa=Generic.x86_64&v=8.2.0_20180214&l=1
                                            ) ## ERROR: (URLError(SSLError(1, u'[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed (_ssl.c:661)'),))

Traceback (most recent call last):
                                              File "/mnt/build_area/LibreELEC.tv/build.LibreELEC-Generic.x86_64-8.2.0_20171113/LibreELEC-settings-33edf50/.install_pkg/usr/share/kodi/addons/service.libreelec.settings/oe.py", line 271, in load_url
                                              File "/usr/lib/python2.7/urllib2.py", line 154, in urlopen
                                              File "/usr/lib/python2.7/urllib2.py", line 429, in open
                                              File "/usr/lib/python2.7/urllib2.py", line 447, in _open
                                              File "/usr/lib/python2.7/urllib2.py", line 407, in _call_chain
                                              File "/usr/lib/python2.7/urllib2.py", line 1241, in https_open
                                              File "/usr/lib/python2.7/urllib2.py", line 1198, in do_open
                                            URLError: <urlopen error [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed (_ssl.c:661)>
```
PKC told Python requests to NOT check SSL certificates (e.g. because you're directly contacting a PMS in your LAN, not via plex.tv). But still the SSL gets checked and fails. Before this is not fixed, don't expect PKC to work like it's supposed to :wink:

**Comment by croneter** [comment]
Should be fixed with new PKC version 2.0.4. Let me know here if this should not be the case.
